README.md

```diff
@@ -40,59 +40,66 @@ The work is available under the Creative Commons [Attribution-NonCommercial-Shar
 | 6    | [Authentication](./week-6/schedule.md)       |
 | 7    | [REST APIs](./week-7/schedule.md)            |
 | 8    | [Full-Stack App](./week-8/schedule.md)       |
```

Comment:
Should we insert Reading Week here as a break? Technically Full-Stack App is Week 9 and React Week is Week 10 right?

Reply:
Interesting question. I am not sure if reading week is going to stay as Week 8. It could shift a week or so in either direction, no? Also, 1 cohort out of 3 it's two weeks. Maybe we should just ignore it for now and treat the first part of the programme as a 9-week unit, regardless of the breaks.

Reply:
Fair enough. I just want to make sure we don't confuse people, since planning that break in advance will be important once they're allowed to actually go do fun stuff like holidays again